### PR TITLE
SGA/DR9 documentation

### PR DESCRIPTION
**Pull request by moustakas**
[WIP] SGA/DR9 documentation.

@geordie666 could you (temporarily) serve this branch at
  http://faraday.uwyo.edu/~admyers/scratch/legacysurvey/output/
so we can review it during the imaging call today?

### CODE REVIEW

**Comment by geordie666** [comment]
@moustakas: I'm now serving this at:

http://faraday.uwyo.edu/~admyers/scratch/legacysurvey/sga/dr9/sga/

(note the slightly different url, i.e., "sga" instead of "output").

File: pages/dr9/sga.rst
Comment (weaverba137) [inline pages/dr9/sga.rst:18]: Why are you using a fixed-width font here?

File: pages/dr9/sga.rst
Comment (weaverba137) [inline pages/dr9/sga.rst:22]: I don't think a pure number has to be rendered as LaTeX math.

File: pages/dr9/sga.rst
Comment (weaverba137) [inline pages/dr9/sga.rst:46]: This is definitely overkill on `:math:`.  It may even be slowing down rendering the page.

**Review by weaverba137 — COMMENTED** [review]
A few things to watch out for.  But maybe this is still a work in progress, so I'll hold off on more in-depth analysis.

**Comment by moustakas** [comment]
@weaverba137 @geordie666 I believe this is ready for your initial review.

Note that I am still working on the files on-disk to match the data model documented here, as well as on the web-app, so please keep that in mind.

**Comment by geordie666** [comment]
I won't be able to review this for a few hours at least. In the meantime, I've served the latest version at

http://faraday.uwyo.edu/~admyers/scratch/legacysurvey/sga/dr9/sga/

**Comment by weaverba137** [comment]
I also have some commitments this afternoon, but hopefully by the end of the day...

File: pages/dr9/sga.rst
Comment (weaverba137) [inline pages/dr9/sga.rst:19]: This would work much better as `20,000 :math:...`

File: pages/dr9/sga.rst
Comment (weaverba137) [inline pages/dr9/sga.rst:42]: In some places you use `:math:grz` and in other places you use `*grz*`.

File: pages/dr9/sga.rst
Comment (weaverba137) [inline pages/dr9/sga.rst:29]: Is there a preference for having the name in a fixed-width font?

File: pages/dr9/sga.rst
Comment (weaverba137) [inline pages/dr9/sga.rst:60]: If this text will never appear, I'd take it out.  Otherwise, add a comment as to why it is commented out.

File: pages/dr9/sga.rst
Comment (weaverba137) [inline pages/dr9/sga.rst:151]: Are these comment lines in between the items deliberate?

File: pages/dr9/sga.rst
Comment (weaverba137) [inline pages/dr9/sga.rst:142]: Since these items are obviously ordered, why not use a numbered list?

File: pages/dr9/sga.rst
Comment (weaverba137) [inline pages/dr9/sga.rst:178]: Why is BYHAND *not* in a fixed-width font?  Are you using backticks in MarkDown style instead of RST style?

File: pages/dr9/sga.rst
Comment (weaverba137) [inline pages/dr9/sga.rst:188]: Similar to the comment above, why not a numbered list that is not so widely spaced out with extraneous comments?

File: pages/dr9/sga.rst
Comment (weaverba137) [inline pages/dr9/sga.rst:207]: Yet another obviously numbered list.  You could do `a.`, `b.` as well.

File: pages/dr9/sga.rst
Comment (weaverba137) [inline pages/dr9/sga.rst:225]: You really don't need math mode here.  Just write `10 arcmin`.

File: pages/dr9/sga.rst
Comment (weaverba137) [inline pages/dr9/sga.rst:230]: You really don't need math mode here.  Just write `93%`.  You have `7%` in the same line!

File: pages/dr9/sga.rst
Comment (weaverba137) [inline pages/dr9/sga.rst:229]: This number does not need to be in math mode.

**Review by weaverba137 — CHANGES_REQUESTED** [review]
There are a lot of formatting things here that are inconsistent with the rest of the legacysurvey.org site as well as with "idoimatic" RST. I also noticed some formatting that is not even consistent within the document itself.  I'm not sure if I should be reviewing the content, because there are still incomplete sections.

File: pages/dr9/sga.rst
Comment (geordie666) [inline pages/dr9/sga.rst:871]: `acknowledgment`, here, is the preferred U.S. spelling (rather than `acknowledgement`). Normally, I wouldn't care about such pedantry. But, here the usage is in a link. This means the link to:
```
../../acknowledgment/#scientific-publication-acknowledgment
```
is broken. I suggest checking all links using the rendered page at

http://faraday.uwyo.edu/~admyers/scratch/legacysurvey/sga/dr9/sga/

**Review by geordie666 — CHANGES_REQUESTED** [review]
@weaverba137 provided a lot of good feedback. I've added one comment about broken links. I'm certainly happy to review the page more extensively once you address the initial feedback from Ben.

File: pages/dr9/sga.rst
Comment (moustakas) [reply pages/dr9/sga.rst:871]: Thanks, noted and updated. But note that the "alternative" spelling appears on our home page (`Acknowledgements`), which is what caused me to adopt this spelling without thinking or double-checking the link(s)--
  https://www.legacysurvey.org/

File: pages/dr9/sga.rst
Comment (moustakas) [reply pages/dr9/sga.rst:42]: There were instances of `:math:grz` in the original draft but none in the version you reviewed.

File: pages/dr9/sga.rst
Comment (moustakas) [reply pages/dr9/sga.rst:19]: I inherited the formatting of this particular bit of text from @geordie666's initial draft (see, e.g., https://github.com/legacysurvey/legacysurvey/blame/master/pages/dr9/sga.rst#L19), which propagated into my using math mode more than I should have.

File: pages/dr9/sga.rst
Comment (moustakas) [reply pages/dr9/sga.rst:188]: I preferred the look of a bulleted list but since this is an issue I've changed this to a numbered list. 

The extra space between the items of the list were because I wanted extra space between each item---the default formatting is far too "scrunched.'

But I don't understand the comment about "extraneous comments". Which portion of the documentation I've written is extraneous and should be removed?

**Comment by moustakas** [comment]
OK, I think I've made all the requested changes, but it's the third-draft of an 800-line document, so I'm sure there are additional errors. 

Some formatting choices I'm still uncertain about are whether columns/quantities like `GROUP_NAME` should have double ticks around them and whether morphologies like `REX` should have double ticks, one tick, or no ticks at all.

Please also review content (if inspired); the only two small sections still to be written are the `Known Issues` and `Future Plans` sections, but those will be bulleted lists that I am going to write after I've worked on the web-app more.

File: pages/dr9/sga.rst
Comment (geordie666) [inline pages/dr9/sga.rst:712]: The second
```
:math:`\chi^{2}`
```
needs to be placed outside of the `*` (italics) clause, or otherwise escaped.

File: pages/dr9/sga.rst
Comment (geordie666) [inline pages/dr9/sga.rst:713]: The `22` in 
```
:math:`\mu=22`
```
needs to be outside of the math environment so that the font is rendered consistently with `22.5, 23 ... etc.`.

**Review by geordie666 — CHANGES_REQUESTED** [review]
I carefully read through the SGA page content and have included some suggestions.

To answer your direct question, both columns/quantities like ``GROUP_NAME`` and morphologies like ``REX`` should have double ticks around them. I've _tried_ to maintain that convention on the rest of the pages (although, I've clearly missed a few cases).

File: pages/dr9/sga.rst
Comment (moustakas) [reply pages/dr9/sga.rst:807]: It's definitely the right link but I saw on the astropy list that "the LEDA server in Lyon has been down for some time" and it appears to still be down now. Let's check again before we publish this page, although I'm not sure what we can do if it's still down.

**Comment by moustakas** [comment]
@geordie666 or @weaverba137 can you take a quick look at this latest commit and tell me what I'm doing wrong? 

All I did was to update the `conf.py` file and move the SGA content to its own subdirectory, but I'm getting an `ERROR 404` when I try to navigate to the sga2020 page (after serving with `nikola auto`, which builds and serves). Thanks.

**Comment by moustakas** [comment]
@geordie666 @weaverba137 wanted to mention that I figured out the `404` issue I was having, above.

@geordie666 there's been a bit of "mission creep" in this PR in that I have now addressed #130, so please take a look. And if you could serve this version of the documentation so it can be reviewed more broadly I'd be grateful.

**Comment by moustakas** [comment]
Ugh. I just reread @djschlegel's email on-list (subject line: "SGA-ellipse versions + missing "external" files in DR9 release") and what I've written in the `external.rst` page does not match what David wrote. I'll update the text, although if we are going to move these external catalogs into the DR9 release directory somewhere we should resolve that question first.

**Comment by weaverba137** [comment]
@moustakas, in addition to all the other stuff to worry about, you should test the updates to the top menu bar on a variety of devices.  We were already very close to spilling over the fixed width of that nav bar.

**Comment by geordie666** [comment]
I've now served the latest pages at:

http://faraday.uwyo.edu/~admyers/scratch/legacysurvey/sga/

@moustakas: It's clear (at least in Firefox) that the toolbar now runs over 2 lines instead of 1 line, so we'll need to remove something else from the toolbar to incorporate "SGA." Perhaps the word "Sky" in "Sky Viewer" so it just reads "Viewer"?

**Comment by geordie666** [comment]
I've tried:

http://faraday.uwyo.edu/~admyers/scratch/legacysurvey/sga/

in Safari, Firefox, and Chrome, and I think the toolbar works, now.

**Comment by moustakas** [comment]
I set up a mirror of the current draft documentation here--
  https://portal.nersc.gov/project/cosmo/temp/ioannis/sga-dr9/

**Comment by geordie666** [comment]
@moustakas: On the recent Zoom call, it looked like there were a handful of "known issues" that you needed to still document. I suggest that you document those and then whenever you're comfortable that any remaining changes are likely to be syntactical or textual (in the short-term), feel free to merge this PR. If there are further textual changes, I can take them up in a later PR.

Please remember to update the `CHANGES.md` file.

**Comment by moustakas** [comment]
@geordie666 apologies for the delay. I should be able to wrap up my final changes by tomorrow or, at the latest, Saturday. 

But I'm nervous about making the SGA stuff live / public in case there are some (documentation) fixes I need to make based on Yao-Yuan's work. Can we merge this branch without actually changing the hosted page so that you can get started on your changes?

Or do you want to branch off this branch so I'm not holding you back?

**Comment by geordie666** [comment]
@moustakas: It's fine to wait. My comment was an invitation to merge this early, if you'd like, but you don't have to. As you noted, we won't serve a new version of the pages until the SGA is ready.

There are a handful of updates I will need to make between you merging this branch and Ben serving the new version of the pages. But I expect that I can make most of those changes in just a few hours. So, the workflow will be:

- Merge this PR whenever you're comfortable with a version of the SGA documentation that we could serve.
- I will make a few more changes in another PR, hopefully withing a day or so of you merging.
- Then Ben can serve the pages.

**Comment by moustakas** [comment]
OK, merging now as recommended. Additional updates (based on the last few checks of the SGA data products) will be made in a separate branch before the updates go "live."

**Comment by moustakas** [comment]
@geordie666 shall we take this opportunity to rename the main branch `main` (from `master`)?

**Comment by geordie666** [comment]
@moustakas: Sure, go ahead and rename `master` if you like.

**Comment by weaverba137** [comment]
@moustakas, please do not do this unless you really know what you are doing _and_ are following GitHub's guide to the process.

**Pull request merged**
